controllers/openstackvmset_controller.go

```diff
@@ -783,7 +783,7 @@ func (r *OpenStackVMSetReconciler) vmCreateInstance(
 	osNetBindings map[string]ospdirectorv1beta1.AttachType,
 ) error {
 
-	evictionStrategy := virtv1.EvictionStrategyLiveMigrate
+	evictionStrategy := virtv1.EvictionStrategy(instance.Spec.EvictionStrategy)
```

Comment:
could default to LiveMigrate only when storage is RWX

Reply:
updated